docs/remasters/15plus25/index.md

```diff
@@ -2,4 +2,7 @@
 
 ## General documentation
 
-* [Editions and updates](updates.md)
\ No newline at end of file
+* [Editions and updates](updates.md)
+## Kingdom Hearts 2.5 HD ReMIX
```

Comment:
Same here: we need a space between that text and the beginning of the next paragraph

Reply:
Anything that improves readability is good to me.